docs/src/pages/layout/breakpoints/breakpoints.md

```diff
@@ -146,6 +166,49 @@ class MyComponent extends React.Component {
 export default withWidth()(MyComponent);
 ```
 
+### unstable_useWidth([options]) => Breakpoint
+
+#### Arguments
+
+1. `options` (*Object* [optional]):
+  In order to perform the server-side rendering reconciliation, we need to render twice.
+  A first time with nothing and a second time with the children.
+  This double pass rendering cycle comes with a drawback. The UI might blink.
+  You can set this flag to `true` if you are not doing server-side rendering.
+  - `options.initialWidth` (*Breakpoint* [optional]):
+  As `window.innerWidth` is unavailable on the server,
+  we default to returning undefined during the first mount.
+  In some situation, you might want to use an heuristic to approximate
+  the screen width of the client browser screen width.
+  For instance, you could be using the user-agent or the client-hints.
+  https://caniuse.com/#search=client%20hint, we also can set the initial width
+  globally using [`custom properties`](/customization/themes/#properties) on the theme.
+  In order to set the initialWidth we need to pass a custom property with this shape:
+  - `options.resizeInterval` (*Number* [optional]): Defaults to 166, corresponds to 10 frames at 60 Hz. Number of milliseconds to wait before responding to a screen resize event.
+  - `options.width` Bypass the width calculation logic and return a set width.
```

Comment:
I don't see any use case for that. Thoughts?

Reply:
I just replicated what was already there, I doubt this would ever be used. I'll remove it